src/content/effects.js

```diff
@@ -11,55 +11,71 @@ import {
 import { setItem } from '../utility/setItem';
 import { getRandomKey } from '../utility/getRandomKey';
 import { items } from './items';
-import { getSkillInDom } from '../utility/getSkillInDom';
+import { setSkill } from '../utility/setSkill';
 import { resetStatus } from '../utility/resetStatus';
-import { initialStatus } from './initialStatus';
+import { changeLocation } from '../utility/changeLocation';
+import { cutInventory } from '../utility/cutInventory';
+import { getRandomNumberBetweenTwo } from '../utility/getRandomNumberBetweenTwo';
+
 
 const effects = {
-  getItem: ([[min, max], specificItem]) => {
-    const quantity = min + Math.random() * (max + 1 - min);
-    for (let i = 1; i <= quantity; i += 1) {
-      let item = getRandomKey(items);
-      while (item.specific) {
-        item = getRandomKey(items);
-      }
-      if (specificItem) {
-        item = items[specificItem];
-      }
-      playerStatus.inventory.push(item.name);
-      setItem(item);
-      while (playerStatus.inventory.length > 15) {
-        playerStatus.inventory.pop();
-        inventoryHolder.removeChild(inventoryHolder.lastChild);
+  getItems: ({ ...payload }) => {
+    Object.entries(payload).forEach(([key, value]) => {
+      const quantity = Array.isArray(value)
+        ? getRandomNumberBetweenTwo(...value)
```

Comment:
Do you understand that this function returns not an integer value? You can add `Math.floor()` to get integers.